File: template/go/pkg/db/mongo.go
package db

import (
	"context"
	"os"

	"go.mongodb.org/mongo-driver/mongo"
	"go.mongodb.org/mongo-driver/mongo/options"
)

func MongoDB() (*mongo.Database, error) {
	cli, err := mongo.Connect(context.Background(), options.Client().ApplyURI(os.Getenv("BOT_FAAS_MONGODB_URI")))
	if err != nil {
		return nil, err
	}
	return cli.Database(os.Getenv("BOT_FAAS_MONGODB_DBNAME")), nil
}

File: template/php/pkg/Db/Mongo.php
<?php

namespace Pkg\Db;


use MongoDB\Client;
use MongoDB\Database;

class Mongo
{
    public static function MongoDB(): Database
    {
        $db = new Client(getenv('BOT_FAAS_MONGODB_URI'));
        return $db->selectDatabase(getenv('BOT_FAAS_MONGODB_DBNAME'));
    }

}
